ecosystem/platform/server/app/controllers/it1_profiles_controller.rb

```diff
@@ -43,6 +43,7 @@ def create
       log @it1_profile, 'created'
       validate_node(v, do_location: true)
       if @it1_profile.validator_verified?
+        current_user.maybe_send_ait1_registration_complete_email
```

Comment:
I know we say "Step 1" and "Step 2" but I guess they can be completed in any order?

Reply:
if someone is KYC exempt- they only need to do step 1. If someone edits their node registration, they'll have to re-verify it and they'll end up getting an email again